torch/lib/THNN/generic/VolumetricAdaptiveAveragePooling.c

```diff
@@ -0,0 +1,302 @@
+#ifndef TH_GENERIC_FILE
+#define TH_GENERIC_FILE "generic/VolumetricAdaptiveAveragePooling.c"
+#else
+
+#define START_IND(a,b,c) (int)floor((float)(a * c) / b)
+#define END_IND(a,b,c) (int)ceil((float)((a + 1) * c) / b)
+// #define START_IND(a,b,c) a * c / b
+// #define END_IND(a,b,c)  (a + 1) * c / b + ((a + 1) * c % b > 0)?1:0
+
+static void THNN_(VolumetricAdaptiveAveragePooling_updateOutput_frame)(
+          real *input_p,
+          real *output_p,
+          long nslices,
+          long isizeT,
+          long isizeW,
+          long isizeH,
+          long osizeT,
+          long osizeW,
+          long osizeH,
+          long istrideT,
+          long istrideW,
+          long istrideH,
+          long strided)
+{
+  long k;
+#pragma omp parallel for private(k)
+  for (k = 0; k < nslices; k++)
+  {
+    /* loop over output */
+    long ti, i, j;
+    for(ti = 0; ti < osizeT; ti++)
```

Comment:
give "ti" a different name; from the name I'd expect it to loop over the time dimension of the input ("ot" doesn't seem unreasonable.